src/entity/user.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+class User {
```

Comment:
```suggestion
export class User {
```
Não costumamos utilizar o `export default`, porque ai se for importar de outro arquivo seria possível importar com outro nome, o que não queremos pois em geral acaba atrapalhando no entendimento e manutenção do código